src/service/resources-impl.js

```diff
@@ -660,6 +723,18 @@ export class Resources {
   }
 
   /**
+   * @param {number} relayoutTop
+   * @private
+   */
+  setRelayoutTop_(relayoutTop) {
+    if (this.relayoutTop_ == -1) {
+      this.relayoutTop_ = relayoutTop;
```

Comment:
The "pass" remeasures all elements under this one. This is what we use for height changes to update siblings.